telegram/bot.py

```diff
@@ -4836,13 +4837,19 @@ def create_new_sticker_set(
                 .. versionchanged:: 13.2
                    Accept :obj:`bytes` as input.
             tgs_sticker (:obj:`str` | `filelike object` | :obj:`bytes` | :class:`pathlib.Path`, \
-                optional): TGS animation with the sticker,
-                uploaded using multipart/form-data. See
-                https://core.telegram.org/animated_stickers#technical-requirements for technical
+                optional): **TGS** animation with the sticker, uploaded using multipart/form-data.
+                See https://core.telegram.org/stickers#animated-sticker-requirements for technical
                 requirements.
 
                 .. versionchanged:: 13.2
                    Accept :obj:`bytes` as input.
+            webm_sticker (:obj:`str` | `filelike object` | :obj:`bytes` | :class:`pathlib.Path`, \
```

Comment:
```suggestion
            webm_sticker (:obj:`str` | :term:`file object` | :obj:`bytes` | :class:`pathlib.Path`, \
```
same everywhere wher you *added* this  - just makes it easier to integrate the changes into v14

Reply:
this generates warnings when building. (term not in glossary). Ignore them?

Reply:
ah, because we're not using intersphinx on master yet. yea, I'm okay with ignoring